### PR TITLE
fix: mouse cell selection with active editor

### LINKED ISSUE #1380: Mouse multi cell selection does not work with active editor
### Describe the bug

As we recently found out in Example19, while an Editor is open we cant select multiple cells with the mouse. it works with the keyboard shift + arrow navigation though.

I guess its the editor stealing the focus but I'll investigate and provide a PR

### Reproduction

* example19
* activate editor in a cell
* try selecting it plus a few neighbors using the mouse

### Which Framework are you using?

Vanilla / Plain JS

### Environment Info

```shell
| Executable          | Version |
| ------------------- | ------- |
| (framework used)    | VERSION |
| Slickgrid-Universal | latest |
| TypeScript          | VERSION |
| Browser(s)          | VERSION |
| System OS           | VERSION |
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/slickgrid-universal/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [Wikis](https://github.com/ghiscoding/slickgrid-universal/wiki).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/slickgrid-universal/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/slickgrid-universal/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### PR DESCRIPTION
**Pull request by zewa666**
this fixes errors preventing to do a multi cell selection while an editor is opened.

Closes #1380

### CODE REVIEW

File: packages/common/src/extensions/__tests__/slickCellRangeSelector.spec.ts [hunk 1]
```diff
@@ -701,4 +705,52 @@ describe('CellRangeSelector Plugin', () => {
       done();
     }, 7);
   });
+
+  it.only('should maintain propagation only if editor is on current cell', () => {
```
Comment (ghiscoding) [inline packages/common/src/extensions/__tests__/slickCellRangeSelector.spec.ts:709]: oops looks like you forgot to remove `.only` after you're done 😉

Comment (zewa666) [reply packages/common/src/extensions/__tests__/slickCellRangeSelector.spec.ts]: yeah thats super unfortunate ... I've added eslint-plugin-jest to detect those. Than started to fix the main ones by adjusting the the eslintignore file. As soon as I touched __tests__ though, ~250 errors came up. So those should be handled in a separate shot not to mess up this PR

Comment (ghiscoding) [reply packages/common/src/extensions/__tests__/slickCellRangeSelector.spec.ts]: hmm yeah that's just linting cosmetic though, it should indeed be better to add ESLint stuff as a separate PR

**Comment by zewa666** [comment]
ok I think I got it set now. so what I've essentially changed is to apply previous behaviors only if the selected/iterated cell isn't the active editor cell. That means you still can't do a multi-select starting from the editor cell but it works for all others.

**Comment by ghiscoding** [comment]
> ok I think I got it set now. so what I've essentially changed is to apply previous behaviors only if the selected/iterated cell isn't the active editor cell. That means you still can't do a multi-select starting from the editor cell but it works for all others.

that seems fine by me, I'm assuming most of the time it will be from another cell anyway. Thanks for the fix :)

**Pull request merged**